Src/Plugins/Tests.Security/EndToEndTests.cs

```diff
@@ -147,7 +151,7 @@ protected void RunAllTests()
                     Path.GetFileNameWithoutExtension(testFileName) + ".sarif";
             }
 
-            RunTest(inputFiles, expectedOutputResourceMap, enforceNotificationsFree: true);
+            RunTest(inputFiles, expectedOutputResourceMap, enforceNotificationsFree: true, parameter: inputFiles);
```

Comment:
>parameter: inputFiles [](http://example.com/codeflow?start=91&length=21)

This will be used in ConstructTestOutputsFromInputResources to get the name of the file under analysis.